frontend/src/libs/components/card/styles.module.scss

```diff
@@ -71,6 +79,10 @@
   word-break: break-word;
 }
 
+.icon-right:hover {
+  background-color: var(--blue-300);
```

Comment:
this is still not fixed, icon color should change not background color

Reply:
@Liza-Veis i just forgot to remove this line. All logic for changick color is in button component